web-services/query/src/test/java/datawave/webservice/query/logic/composite/CompositeQueryLogicTest.java

```diff
@@ -372,9 +442,9 @@ public void setup() {
     
     @Test
     public void testClone() throws Exception {
-        List<QueryLogic<?>> logics = new ArrayList<>();
-        logics.add(new TestQueryLogic());
-        logics.add(new TestQueryLogic());
+        Map<String,QueryLogic<?>> logics = new HashMap<>();
+        logics.put("TestQueryLogic", new TestQueryLogic());
+        logics.put("TestQueryLogic2", new TestQueryLogic());
```

Comment:
These names should probably be static constants

Reply:
no need as this is a test case.